src/components/home/modals/TokenSuccess.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <div>
+    <div class="token-succ-icon">
+      <img v-if="txType==='Destroy Token'"
+           src="../../../assets/imgs/icons/operate/ic_burn.svg">
+      <img v-else
+           src="../../../assets/imgs/icons/operate/ic_success_circle.svg">
+    </div>
+    <div class="infos">
+      <b-form-group v-if="txType==='Destroy Token'"
+                    horizontal
+                    class="form-line"
+                    label="Burn Amount"
+                    label-for="amount_success">
+        <b-form-input id="amount_success"
+                      :value="formatter(amount)"
+                      class="amount"
+                      readonly
+                      :plaintext="true">
+        </b-form-input>
+      </b-form-group>
+      <b-form-group v-else-if="txType==='Register New Token'"
```

Comment:
confuse about the txType `Register New Token`? register contract or issue token? and what Total Supply mean in line 25?

Reply:
This is register contract. And Line 25 is changed to Max Supply.